projects/admin/src/app/routes/items-route.ts

```diff
@@ -87,8 +102,10 @@ export class ItemsRoute extends BaseRoute implements RouteInterface {
    * @param record - object, record to be saved
    */
   private getUrl(record: any) {
-    const pid = extractIdOnRef(record.metadata.document.$ref);
-    return of(`records/documents/detail/` + pid);
+    const redirectTo = this._routeToolService.getRouteQueryParam('redirectTo');
+    return of((redirectTo)
+        ? redirectTo
+        : `records/documents/detail/` + extractIdOnRef(record.metadata.document.$ref));
```

Comment:
why `` here? Replace by '' or put extractIdOneRef in a variable.

Reply:
done